src/main/java/org/commcare/formplayer/objects/MediaMetadataRecord.java

```diff
@@ -0,0 +1,87 @@
+package org.commcare.formplayer.objects;
+
+import lombok.Getter;
+
+import lombok.Setter;
+import org.commcare.formplayer.util.Constants;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import java.time.Instant;
+
+/**
+ * Model class representing a postgres entry for media metadata table
+ */
+@Entity
+@Table(name = Constants.POSTGRES_MEDIA_META_DATA_TABLE_NAME)
+@Getter
+public class MediaMetadataRecord {
+
+    @Id
+    @GeneratedValue(generator = "uuid")
+    @GenericGenerator(name = "uuid", strategy = "org.hibernate.id.UUIDGenerator")
+    private String id;
```

Comment:
@shubham1g5 is this ID going to be generated in the DB or manually?

Reply:
This gets [generated manually](https://github.com/dimagi/formplayer/blob/saveMediaAttachment/src/main/java/org/commcare/formplayer/services/MediaHandler.kt#L24) , so we should not need to specify generator annotations here.